bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -177,6 +176,11 @@ where
 		}))
 	});
 
+	io.add_sync_method("bitacross_getScheduledEnclaveList", move |_: Params| {
+		debug!("worker_api_direct rpc was called: bitacross_getScheduledEnclaveList");
+		Ok(json!({ "signer": format!("{:?}", &GLOBAL_SCHEDULED_ENCLAVE.current_mrenclave) }))
```

Comment:
This is a `RwLock`, you have to first obtain the `read` lock and then try to get the value. Reference - https://github.com/litentry/litentry-parachain/blob/d893864ee3dbded7d05e9c7921936a30c58841c5/bitacross-worker/litentry/core/scheduled-enclave/src/lib.rs#L51C44-L52C1

Reply:
Use this function instead `get_current_mrenclave()`, reference - https://github.com/litentry/litentry-parachain/blob/d893864ee3dbded7d05e9c7921936a30c58841c5/bitacross-worker/litentry/core/scheduled-enclave/src/io.rs#L134